packages/react-components/package.json

```diff
@@ -13,7 +13,8 @@
         }
     },
     "files": [
-        "./dist"
+        "./dist",
+        "./src"
```

Comment:
Why add this? Looks wrong to me.

Reply:
To support src maps. Otherwise there's no point to having src maps included in the build. If we include src, editors and debuggers pick it up to give developers a better experience.